components/ComboBox/ComboBox.constants.ts

```diff
@@ -1,8 +1,4 @@
-export interface Option {
-  content: string
-  id: string
-}
-
+import { Option } from './ComboBox.types'
 export const options: Option[] = [
```

Comment:
don't use constant variables in components, get data outside components as props